scripts/apps/ingest/controllers/IngestListController.js

```diff
@@ -17,6 +17,7 @@ export class IngestListController extends BaseListController {
             $scope.loading = true;
             criteria.aggregations = 1;
             criteria.es_highlight = search.getElasticHighlight();
+            criteria.source.size = 50;
```

Comment:
I wonder if this could be done differently, so that we don't have it in two places...